src/test/java/com/evanlennick/retry4j/RetryConfigBuilderTest_ValidationDisabled.java

```diff
@@ -73,7 +73,7 @@ public void testSettingDurationBetweenTries_seconds() {
     @Test
     public void testSettingDurationBetweenTries_millis() {
         RetryConfig config = retryConfigBuilder
-                .withDelayBetweenTries(5000L)
+                .withDelayBetweenTries(5, ChronoUnit.MILLIS)
                 .build();
 
         assertThat(config.getDelayBetweenRetries().toMillis()).isEqualTo(5000);
```

Comment:
Not sure if you want to keep this (and even the `_seconds`) test since now the `withDelayBetweenTries` method deals strictly with `Duration` directly before setting it in the `RetryConfig`.